packages/terra-status-view/docs/README.md

```diff
@@ -1,32 +1,13 @@
 # Terra Status View
 
-The status view component provides a customizable icon, message, and buttons in a view.
+Renders an icon, title, message, and buttons.
```

Comment:
This doesn't really tell consumers anything about what this component is or what it should be used for.

Reply:
updated